### ISSUE TITLE
Accessibility features 

### ISSUE THREAD

**Issue by juhahakala** [dcmi/dc-srap#24]
According to The Web Accessibility Directive (https://digital-strategy.ec.europa.eu/en/policies/web-accessibility) Internet resources must be accessible. Description of available accessibility features (or lack thereof) is important for users who need accessible documents. MARC 21 and ONIX enable description of accessibility features or deficiencies. In Dublin Core, this information can only be provided in Description, which limits the usefulness of this metadata and prevents accurate migration to other metadata. 

Adding the following term to DCMIMT is proposed: 

Accessibility features   

textual information describing the accessibility features of a resource, including technical details

**Comment by juhahakala** [comment]
Added as Accessibility statement, with the following description: 

Textual information describing the accessibility features of a resource, including technical details.

Recommended practice is to define the accessibility options, such as software requirements for using the document or technical features such as open or closed captioning.
